rules/aar_import/impl.bzl

```diff
@@ -344,7 +344,7 @@ def _process_jars(
         ),
         source_jar = source_jar,
         neverlink = False,
-        deps = r_java_info + java_infos,  # TODO(djwhang): Exports are not deps.
+        deps = java_infos,  # TODO(djwhang): Exports are not deps.
```

Comment:
This line specifically is causing the internal test to fail. I will follow up with internal teams to chase down the root cause.

Reply:
reverted. the java_infos being added as deps were exposing r classes as runtime deps which is not correct and was causing issues in our repo. Instead I set r java infos as neverlink.